.github/workflows/tests.yaml

```diff
@@ -0,0 +1,25 @@
+name: Run tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Install test requirements
+        run: |
+          pip install -r test-requirements.txt
+
+      - name: Run tests
```

Comment:
Looks like this is failing due to:

> /home/runner/work/workflows/workflows/scripts/install-kustomize.sh: line 7: BINDIR: unbound variable

Likely need to add an env hash similar to the `validate-manifests.yaml` workflow:
```
    env:
      BINDIR: "${{ github.workspace }}/bin"
```

Reply:
Yup, I made some last minute changes and didn't re-run the tests :). Yay CI!